rocAL/rocAL/source/master_graph.cpp

```diff
@@ -659,12 +659,12 @@ MasterGraph::copy_out_tensor(void *out_ptr, RaliTensorFormat format, float multi
             auto img_buffer = out_image;
             if (format == RaliTensorFormat::NHWC)
             {
```

Comment:
please revert the changes in this file since nothing has changed

Reply:
reverted